modules/caddyhttp/reverseproxy/fastcgi/fastcgiclient/client.go

```diff
@@ -21,7 +21,7 @@
 // Use of this source code is governed by a BSD-style
 // Part of source code is from Go fcgi package
 
-package fastcgi
+package fastcgiclient
```

Comment:
Totally bike shedding this but I wonder if `fcgiclient` would be a little more conventional to mirror the `net/http/fcgi` package.

Reply:
Implemented your suggestion to rename the package from `fastcgiclient` to `fcgiclient` for consistency with the net/http/fcgi package.